pinot/tests/test_metrics.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+import numpy as np
+import numpy.testing as npt
+import torch
+
+def test_import():
+    import pinot.metrics
+
+def test_mse_and_rmse():
+    import pinot
+    
+    # we only test here that:
+    #  * the implementation is consistent with numpy
+    #  * rmse is root of mse
+    for _ in range(5):
+        x = np.random.normal(size=(10,))
+        y = np.random.normal(size=(10,))
+
+        npt.assert_almost_equal(
```

Comment:
When there are many tests in a loop, can use pytest to generate many tests whose success / failure are all reported https://docs.pytest.org/en/latest/parametrize.html#parametrize , rather than exiting the loop the first time an assertion fails